src/FFmpegReader.cpp

```diff
@@ -1095,7 +1112,12 @@ bool FFmpegReader::GetAVFrame() {
 #if IS_FFMPEG_3_2
 		frameFinished = 0;
 
-		ret = avcodec_send_packet(pCodecCtx, packet);
+		if (HasAlbumArt()) {
+			ret = avcodec_send_packet(pCodecCtx, &(pFormatCtx->streams[videoStream]->attached_pic));
+		}
+		else {
+			ret = avcodec_send_packet(pCodecCtx, packet);
+		}
```

Comment:
Decoding works both with and without this code, but I need to test if this brings any performance changes.